job/nullable_int.go

```diff
@@ -0,0 +1,51 @@
+package job
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+// NullableInt represents an int that can be "empty" or nil.
+//
+// A NullableInt is empty by default.
+type NullableInt struct {
+	hasValue bool // false by default
+	i        int
+}
+
+// IsNil returns true if the value is empty.
+func (ni *NullableInt) IsNil() bool {
+	return !ni.hasValue
+}
+
+// Int returns the value, make sure to check ISNil before using Int.
+func (ni *NullableInt) Int() int {
+	return ni.i
+}
+
+// Set sets the value to i.
+func (ni *NullableInt) Set(i int) {
```

Comment:
Likewise, this should be renamed to `SetValue` for consistency

Reply:
I'm not sure about that... since `NullableInt` is actually a value, `SetValue` seems a bit redundant.